api/src/main/java/org/pmiops/workbench/cdr/CdrDataSource.java

```diff
@@ -70,31 +69,9 @@ void resetTargetDataSources() {
     for (DbCdrVersion cdrVersion : getCdrVersions()) {
       try {
         DataSource dataSource = createCdrDs(cdrVersion.getCdrDbName());
-        if (dataSource instanceof org.apache.tomcat.jdbc.pool.DataSource) {
-          org.apache.tomcat.jdbc.pool.DataSource tomcatSource =
-              (org.apache.tomcat.jdbc.pool.DataSource) dataSource;
-          // A Tomcat DataSource implements PoolConfiguration, therefore these pool parameters can
-          // normally be populated via @ConfigurationProperties. Since we are directly initializing
-          // DataSources here without a hook to @ConfigurationProperties, we instead need to
-          // explicitly initialize the pool parameters here. We override the primary connection
-          // info, as the autowired PoolConfiguration is initialized from the same set of properties
-          // as the workbench DB.
-          PoolConfiguration cdrPool = new PoolProperties();
-          BeanUtils.copyProperties(basePoolConfig, cdrPool);
-          cdrPool.setUsername("workbench"); // consistent across environments
-          cdrPool.setPassword(getEnvRequired("WORKBENCH_DB_PASSWORD"));
-          cdrPool.setUrl(String.format("jdbc:mysql:///%s", cdrVersion.getCdrDbName()));
-          tomcatSource.setPoolProperties(cdrPool);
-
-          // The Spring autowiring is a bit of a maze here, log something concrete which will allow
-          // verification that the DB settings in application.properties are actually being loaded.
-          log.info("using Tomcat pool for CDR data source, with minIdle: " + cdrPool.getMinIdle());
-        } else {
-          log.warning(
-              "not using Tomcat pool or initializing pool configuration; "
-                  + "this should only happen within tests");
```

Comment:
Removing for now. The improved logging should help us avoid this misconfiguration.